Sources/SwifterSwift/Foundation/NSUbiquitousKeyValueStoreExtensions.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  NSUbiquitousKeyValueStoreExtensions.swift
+//  SwifterSwift
+//
+//  Created by Joseph Canale on 6/7/19.
+//  Copyright © 2017 SwifterSwift
+//
+
+import Foundation
+#if canImport(Foundation) && !os(Linux)
```

Comment:
Does this build for `watchOS`? According to the documentation it's not available there.

Reply:
The tests built and ran I think.  Well the CI/CD pipeline didn't fail.  I added `!os(watchOS)` to the top here, just to be sure.